build.gradle

```diff
@@ -14,6 +14,7 @@ apply from: "$rootDir/gradle/deploy.gradle"
 version = '1.6.10-SNAPSHOT'
 group = "com.github.spotbugs"
 def spotBugsVersion = '3.1.11'
+def slf4jVersion = '1.8.0-beta2'
```

Comment:
I think we need to seriously consider dropping SLF4J and just using JUL, for maximum compatibility.

Reply:
If you mean the trouble around gradle plugin, I think all the reported issues can be solved if we stop depending on Gradle internal API that excludes SLF4J from the dependency. So I still think that SLF4J is better than that horrible JUL...

Reply:
+1 for JUL.. it just works.